packages/manager/modules/manager-components/src/resource-selector/component.js

```diff
@@ -0,0 +1,14 @@
+import template from './template.html';
+
+export default {
+  bindings: {
+    currencySymbol: '@?',
```

Comment:
is this not available in price? price will have price text which has currency symbol. We need to avoid taking currencySymbol as input. If required you can parse price and get it.

Reply:
This has been removed. The user will have to pass a string for the price, that includes the currency as well